src/helm/proxy/models.py

```diff
@@ -125,6 +125,31 @@ def engine(self) -> str:
         description="Jurassic-1 Large (7.5B parameters)",
         tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG, AI21_TOKENIZER_TAG],
     ),
+    # AI21 Jurassic-2 Models: https://www.ai21.com/blog/introducing-j2
```

Comment:
Just to double check, is it the same tokenizers?

Reply:
Asked the AI21 folks; it is indeed the same tokenizers and the same sequence length.